ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -100,6 +107,18 @@ ORK_CLASS_AVAILABLE
                                           delegate:(id<ORKPasscodeDelegate>)delegate;
 
 /**
+ @param forgotPasscodeText text to display for forgot passcode button
+ 
+ @param text        The message displayed to the user.
+ @param delegate    The delegate for the passcode view controller.
+ 
+ @return A passcode authentication view controller.
+ */
++ (id)passcodeAuthenticationViewControllerWithForgotPasscodeText:(nullable NSString*)forgotPasscodeText
```

Comment:
Elsewhere in RK, the delegate responds optionally to three methods:

1. hasForgotPasscode
2. textForForgotPasscode
3. forgotPasscode tapped

Rather than using a special initializer. Would like to stick to that convention. Thanks.